util/docker/docker_cleanup_test.go

```diff
@@ -0,0 +1,57 @@
+package docker
+
+import (
+	"context"
+	"io"
+	"io/ioutil"
+	"testing"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/filters"
+	"github.com/docker/docker/api/types/volume"
+	"github.com/docker/docker/client"
+	"github.com/mongodb/grip"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCleanup(t *testing.T) {
```

Comment:
can we break this into sub tests (depending on how we break up the main function? just so when it errors its clear what has broken?

Reply:
done.